tools/apidiff/merger.cs

```diff
@@ -13,11 +13,14 @@ static string GetVersion (string line)
 		return line.Substring (start, end - start);
 	}
 
-	public static void Process (string platform, string path, string os)
+	public static void Process (string platform, string path, string os, string? destinationPath = null)
```

Comment:
If you are unconditionally calling this with arg[3] or null you don't need the default argument here.